qemu/tests/hotplug_port_chardev_pci_with_console.py

```diff
@@ -0,0 +1,45 @@
+from virttest import error_context
+
+from qemu.tests.virtio_console import add_chardev
+from qemu.tests.virtio_console import add_virtio_ports_to_vm
+from qemu.tests.virtio_serial_file_transfer import transfer_data
+from qemu.tests.virtio_serial_hotplug_port_pci import get_buses_and_serial_devices
+
+
+def get_virtio_serial_pci(vm, serial_device):
+    serial_device_bus = serial_device.get_param('bus')
+    local_index = serial_device_bus.index('.')
+    serial_bus_id = serial_device_bus[:(local_index)]
```

Comment:
`serial_bus_id = serial_device_bus.split('.')[0]`

Reply:
thanks and update and will update test result later.